CONTRIBUTING.md

```diff
@@ -33,7 +38,84 @@ run the following command:
 $ pre-commit run --all
 ```
 
-Once `pre-commit` is set up
-you may commit code and create pull requests as you would expect.
-Pull requests require approval of at least one maintainer
-and [CI to be passing](https://github.com/returntocorp/semgrep-rules/actions).
+Once `pre-commit` is set up you may commit code and create pull requests as you would expect.
+
+### Tests
+
+Test files should be included with new rules. Tests are added by included a source code file with the same name as the rule file. For example, if the rule is in `my-rule.yaml`, the tests should be in `my-rule.js` (or another valid extension for the target language). Inside the test file, expected findings should be marked with a comment that reads `ruleid: my-rule` on the line directly above. You can also include code that should **not** be detected with `ok: my-rule`. Use the appropriate single-line comment syntax for the target language (`#` for Python, `//` for JavaScript, etc.). An example is listed below.
```

Comment:
Why duplicate this information vs. just linking to https://semgrep.dev/docs/writing-rules/testing-rules/?

Reply:
My thinking is:
1) I want this information all on one page.
2) Therefore, I think it will be beneficial for contributors to see it spelled out all on the same page.
3) Also, this isn't as detailed as the `testing-rules` in the docs page. It's just a quick summary to let contributors know that it's expected: see (1) and (2).